### PR TITLE
AdapterList fix

### PR DESCRIPTION
**Pull request by dgomolka**
- fixes incorrect disposal in AdapterList (fast scroll crash)
- added onImageReady callback
- Compose updated to 0.1.0-dev11

### CODE REVIEW

**Comment by dgomolka** [comment]
There still exists an edge case where it can sometimes crash during fast scroll when the list items are still loading for the first time

File: build.gradle [hunk 1]
```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.1.0-alpha03'
+        classpath 'com.android.tools.build:gradle:4.1.0-alpha08'
```
Comment (mvarnagiris) [inline build.gradle]: Have you tried making a build off your commit in jitpack? I found that on `alpha08` META-INF is missing after build and you cannot use this library. Related to [this](https://stackoverflow.com/questions/61450260/extension-function-cant-be-imported-from-library)

**Comment by mvarnagiris** [comment]
This is great! Thanks!

File: build.gradle (hunk 1 above)
Comment (dgomolka) [reply build.gradle]: You're right, I think Gradle Module Metadata file would be needed - https://blog.gradle.org/gradle-metadata-1.0.
Reverted the version for now. But.. The method is not visible for import in an external project, but the symbol is there, compiled (with 4.1.0-alpha08 it's not even accessible/compiled). META-INF is missing also. Do you have any ideas?

**Pull request merged**